packages/core/addon/components/field-renderer.js

```diff
@@ -45,6 +62,12 @@ export default class FieldRenderer extends Component {
     this.args.setFieldName(this.args.field.name, this.newFieldName);
   }
 
+  @action
+  updateFieldLabel(newLabel) {
+    this.newFieldLabel = newLabel;
+    this.args.setFieldLabel(this.args.field.name, this.newFieldLabel);
```

Comment:
@robustdj this is where we fire an action to signal that the label input has changed instead of just directly changing the `Field.label` (since it has a getter only--the way to set a `Field.label` is actually to call `Field.setLabel()` which is the action that the `CardManipulator` component is sending down to us.